TAInformation/views.py

```diff
@@ -125,7 +122,7 @@ def get(self, request):
     # Postcondition: Creates new user, if data entered validates successfully and user doesn’t already exist.
     # Side Effects: Message indicating result is displayed at the bottom of the “Create Courses” form
     def post(self, request):
-        # Extract data from form
+        Extract data from form
```

Comment:
This is a comment for the block of code to follow.  It should be commented out.

Reply:
Fixed